lib.php

```diff
@@ -341,10 +341,19 @@ public function ajax_section_move() {
      *     each of values is an array of block names (for left and right side columns)
      */
     public function get_default_blocks() {
-        return array(
-            BLOCK_POS_LEFT => array(),
-            BLOCK_POS_RIGHT => array('search_forums', 'news_items', 'calendar_upcoming', 'recent_activity')
-        );
+        $use_def_blocks = get_config('format_topcoll', 'defaultdisplayblocks');
```

Comment:
$use_def_blocks is not needed as only used once, so put get_config in 'if' statement.